param/reactive.py

```diff
@@ -252,39 +253,36 @@ def resolve(self):
         else:
             return self._reactive()
 
-    def set_input(self, new):
+    @value.setter
+    def value(self, new):
         """
         Allows overriding the original input to the pipeline.
         """
         if isinstance(self._reactive, Parameter):
-            raise ValueError(
-                "Parameter.rx.set_input() is not supported. Cannot override "
+            raise AttributeError(
+                "`Parameter.rx.value = value` is not supported. Cannot override "
```

Comment:
You can't trace back from the Parameter to its `owner`?

Reply:
Sure, it just seems weird to do that, especially now that we've tightened up the semantics to only update the reactive expression itself.